service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -386,16 +386,30 @@ spec:
     jaeger:
       name: jaeger
       install:
-        storage: Memory # or Elasticsearch
-        memory: {} # details specific to memory storage
-        elasticsearch: {} # details specific to elasticsearch storage
+        storage:
+          type: Memory # or Elasticsearch for production mode
+          memory:
+            maxTraces: 100000
+          elasticsearch: # the following values only apply if storage:type:=Elasticsearch
+            storage: # specific storageclass configuration for the Jaeger Elasticsearch (optional)
+              size: "100G"
+              storageClassName: "storageclass"
+            nodeCount: 3
+            redundancyPolicy: SingleRedundancy
   runtime:
     components:
-      tracing.jaeger: {} # general jaeger specific runtime configuration
-      tracing.jaeger.elasticsearch: {} # runtime configuration for jaeger elasticsearch deployment
+      tracing.jaeger: {} # general Jaeger specific runtime configuration (optional)
+      tracing.jaeger.elasticsearch: #runtime configuration for Jaeger Elasticsearch deployment (optional)
+        container:
+          resources:
+            requests:
+              memory: "1Gi"
+              cpu: "500m"
+            limits:
+              memory: "1Gi"
 ----
 
-The Jaeger installation can be customized with the `install` field. Container configuration, such as resource limits is configured in `spec.runtime.components.jaeger` related fields. If a Jaeger resource matching the value of name exists, the control plane will be configured to use the existing installation. Use an existing Jaeger resource to fully customize your Jaeger installation.
+The Jaeger installation can be customized with the `install` field. Container configuration, such as resource limits is configured in `spec:runtime:components:jaeger:` related fields. If a Jaeger resource matching the value of `spec:addons:jaeger:name:` exists, the control plane will be configured to use the existing installation. Use an existing Jaeger resource to fully customize your Jaeger installation.
```

Comment:
@JStickler Why are we changing the separators from dots to colons? Is that jaeger specific?

Reply:
Nope, that was me getting myself confused.  I've reverted those changes back to what they should be.